src/typeInference/annotateTypeSwitchOperator.ts

```diff
@@ -48,23 +49,22 @@ function checkComparison(
 	argumentType: SequenceType,
 	returnType: SequenceType
 ): SequenceType | undefined {
```

Comment:
This undefined can go right?

Reply:
Nope, I use it in the actual annotation function.